.github/actions/frontend/chromatic/action.yml

```diff
@@ -23,21 +23,12 @@ runs:
         echo "github_token=${{ inputs.github_token }}"
         echo "chromatic_project_token=${{ inputs.chromatic_project_token }}"
         echo "publish_chromatic=${{ fromJSON(inputs.publish_chromatic) }}"
-    - name: chromatic_upload
-      if: ${{ !fromJSON(inputs.publish_chromatic) }}
-      uses: chromaui/action@v1
-      with:
-        token: ${{ inputs.github_token }}
-        projectToken: ${{ inputs.chromatic_project_token }}
-        onlyChanged: true
-        exitOnceUploaded: true
     - name: chromatic_publish
-      if: ${{ fromJSON(inputs.publish_chromatic) }}
       uses: chromaui/action@v1
       with:
         token: ${{ inputs.github_token }}
         projectToken: ${{ inputs.chromatic_project_token }}
         onlyChanged: true
         # Automatically accepts all changes in Chromatic as this was already done in the PR stage.
-        autoAcceptChanges: true
+        autoAcceptChanges: ${{ fromJSON(inputs.publish_chromatic) }}
```

Comment:
@Sivli-Embir Simplified this logic per [this comment](https://github.com/JupiterOne/.github/pull/90#discussion_r1304920711)